README.md

```diff
@@ -91,6 +95,73 @@ Please be aware that the native `source.yml` connection set up in the package wi
 
 To connect your multiple schema/database sources to the package models, follow the steps outlined in the [Union Data Defined Sources Configuration](https://github.com/fivetran/dbt_fivetran_utils/tree/releases/v0.4.latest#union_data-source) section of the Fivetran Utils documentation for the union_data macro. This will ensure a proper configuration and correct visualization of connections in the DAG.
 
+## (Optional) Step 4: Utilizing Workday HCM History Mode records
+
+If you have History Mode enabled for your Workday HCM connector, we now include support for the worker, worker position, worker position organization, and personal information tables directly. You can view these files in the [`staging/workday_history`](https://github.com/fivetran/dbt_workday/blob/main/models/staging/workday_history) folder. This staging data then flows into the employee daily history model, which in turn populates the monthly summary model. This will allow you access to your historical data for these tables for the most accurate record of your data over time.
+
+### IMPORTANT: How To Update Your History Models
+To ensure maximum value for these history mode models and avoid messy historical data that could come with picking and choosing which fields you bring in, **all fields in your Workday HCM history mode connector are being synced into the workday history staging models**.
+
+
+To update the history mode models, you must follow these steps: 
+1) Go to your Fivetran Workday HCM History Mode connector page.
+2) Update the fields that you are bringing into the model. 
+3) Run a `dbt run --full-refresh` on the specific staging models you've updated to bring in these fields and all the historical data available with these fields.
```

Comment:
Removing.

Reply:
If I recall, all of the below steps were only necessary due to the incremental logic (if you add a field the incremental models fail). Since we have no incremental logic, customers can add and subtract fields from the connector as much as they want. If this understanding is correct, we can remove the entire section below:
![image](https://github.com/fivetran/dbt_workday/assets/74217849/4a8f1028-1391-44e5-883e-f25f5ec55ff3)

Reply:
I also believe this section is now pointing to an out of date folder
![image](https://github.com/fivetran/dbt_workday/assets/74217849/4fcabda8-f445-496a-8ce4-dc772f2961d3)

Reply:
Also last topic on these - it looks like we have two "Step 4" sections. Could we correct the ordering or consolidate them. Whichever works best in your opinon.

Reply:
Removed update section, updated folder, corrected steps ordering.